README.md

```diff
@@ -394,6 +394,18 @@ in the Installation Guide.
     or JupyterLab installation, as described in [section
     "Launching SageMath"](https://doc.sagemath.org/html/en/installation/launching.html)
     in the installation manual.
+    
+Alternative Installation using PiPy
+---------------
+
+You can find `sage` and `sagemath` pip packages but it is worth nothing that they are not at all realted to this `sage`. Indeed `pip install sage` installs `sage 0.0.0` which is literally an empty package. For installation of this `sage` you need to install `sagemath-standard`. First activate your python virtual environment and follow these steps:
```

Comment:
I don't think it is needed to talk about these packages.

Reply:
These packages are very misleading during sage installation in a python virtual environment and the sources on the internet do not explain clearly about the installation of sage package using pip.

Reply:
At least don't start the writing with what shouldn't be done. And nobody needs to know in what specific way these packages are unsuitable. As simple "(Ignore the `sage` and `sagemath` packages on PyPI.)" should be enough.

Reply:
Agreed. Thank you

Reply:
Perhaps a note would be better to avoid this ambiguity of `sage` and `sagemath` in PiPy.

Reply:
Perhaps we should contact the maintainer of the empty sage package, @dhagrow, and ask if they would mind transferring the name to the SageMath org somehow (unsure if this ping here does anything but it's fairly easy to find an associated email address for that user on google if we did want to get in conact if desired).

Reply:
That's #28796.